test/programmatic_model_creation/programmatic_model_expansion.jl

```diff
@@ -58,6 +62,7 @@ let
 end
 
 # Tests creating a network and adding reactions.
+# This test seems weird?
```

Comment:
```suggestion
```
Not really something to comment on in the test. Add a PR comment on Github if you have questions about a test. This is just checking that we end up with the right number of parameters and states. (But will be deleted anyways so doesn't matter...)